src/tests/functional/plugin/gpu/concurrency/gpu_concurrency_tests.cpp

```diff
@@ -108,3 +108,83 @@ INSTANTIATE_TEST_SUITE_P(smoke_RemoteTensor, OVConcurrencyTest,
     ::testing::Combine(::testing::ValuesIn(num_streams),
         ::testing::ValuesIn(num_requests)),
     OVConcurrencyTest::getTestCaseName);
+
+TEST(OVConcurrencyTest, canSwapTensorsBetweenIRs) {
+    std::vector<std::vector<uint8_t>> ref;
```

Comment:
can we test this on 2 dGPU env? from my previous test, swap buffer will lead to hang, not sure if same root cause as this issue.

Reply:
I have added a couple of new tests and checked it on DG1+DG1+iGPU system, looks like it works good